raven/processes/wps_nalcms_zonal_stats.py

```diff
@@ -152,30 +159,36 @@ def _handler(self, request, response):
             categories = SIMPLE_CATEGORIES
         else:
             categories = TRUE_CATEGORIES
+        summary_stats = SUMMARY_ZONAL_STATS
 
         try:
             stats = zonal_stats(
-                projected, raster_file, stats=['count', 'nodata', 'nan'],
+                projected, raster_file, stats=summary_stats,
                 band=band, categorical=True, all_touched=touches,
                 geojson_out=True, raster_out=False)
 
-            lu = defaultdict(lambda: 0)
+            land_use = list()
             for stat in stats:
+                lu = defaultdict(lambda: 0)
                 prop = stat['properties']
 
                 # Rename land-use categories
                 for k, v in categories.items():
                     lu[v] += prop.pop(k, 0)
+                for k in summary_stats:
+                    lu[k] = prop.pop(k, 0)
```

Comment:
I don't like the idea of mixing the land-use count with other stats. To me these are two different objects. Also, then you need to jump through hoops to find the total number of pixels, because you need to exclude those stats. 
You want to be able to do `sum(lu.values())` and get the total number of pixels.

Reply:
Also, I thought we were going to aggregate the land use from all features in the output. Here you are passing a list.

Reply:
That's fair and an easy change. We're saying that we don't want to save the nodata and/or nan values in the landuse data either?

Reply:
You've got a good point there. I suppose nodata and nan's have the same meaning as the other land-use entries, they count a fraction of the pixels. `count` would be out however. Does that make sense ?

Reply:
From what i can tell, `nodata` and `nan` pixels do not count toward the `count` total. They are currently saved in the GeoJSON but they have no bearing on the most recent `land-use` outputs.

Reply:
Then let's keep them out of the lu.